.github/workflows/ci.yml

```
@@ -49,68 +49,27 @@ jobs:
               "ansible-version": "stable-2.9"
             },
             {
-              "ansible-version": "stable-2.12",
-              "python-version": "3.7"
+              "ansible-version": "stable-2.12"
             },
             {
-              "ansible-version": "stable-2.12",
-              "python-version": "3.11"
+              "ansible-version": "stable-2.13"
             },
             {
-              "ansible-version": "stable-2.13",
               "python-version": "3.7"
             },
             {
-              "ansible-version": "stable-2.13",
-              "python-version": "3.11"
-            },
-            {
-              "ansible-version": "stable-2.14",
-              "python-version": "3.7"
-            },
-            {
-              "ansible-version": "stable-2.14",
               "python-version": "3.8"
             },
             {
-              "ansible-version": "stable-2.14",
-              "python-version": "3.11"
-            },
-            {
-              "ansible-version": "stable-2.15",
-              "python-version": "3.7"
-            },
-            {
-              "ansible-version": "stable-2.15",
-              "python-version": "3.8"
-            },
-            {
-              "ansible-version": "stable-2.15",
-              "python-version": "3.11"
-            },
-            {
-              "ansible-version": "milestone",
-              "python-version": "3.7"
-            },
-            {
-              "ansible-version": "milestone",
-              "python-version": "3.8"
-            },
-            {
-              "ansible-version": "milestone",
               "python-version": "3.9"
             },
             {
-              "ansible-version": "devel",
-              "python-version": "3.7"
-            },
-            {
-              "ansible-version": "devel",
-              "python-version": "3.8"
+              "ansible-version": "stable-2.14",
+              "python-version": "3.12"
```

```suggestion
              "python-version": "3.12"
```

What about excluding python 3.12 in general?